### DISCUSSION TITLE
My revanced on android 7 has been stopped playing and keeps buffering on 1:09 or so

### DISCUSSION THREAD

**Discussion by stresslicious** [kitadai31/revanced-patches-android6-7#41] [Q&A]
Hey, anyone know how to fix revanced that keeps buffering? I've been using this version of revanced for my android 7, and it works smoothly for few months. But--suddenly today it keeps buffering on 1:09 or so. I've tried to deleted my data cache on app settings, turn on and off that spoof thing, also turn on and off v9 thing. It just doesn't work. Helppp

**Comment by kitadai31** [comment]
Already fixed in [v2.160.5](https://github.com/kitadai31/revanced-patches-android6-7/releases/tag/v2.160.5)
